monitor.cpp

```diff
@@ -5,34 +5,40 @@
 #include <iostream>
 using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
 
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if (temperature > 102 || temperature < 95) {
-    cout << "Temperature is critical!\n";
+void sleep() {
```

Comment:
The name of this function - `sleep` goes by the code inside it.
Try naming the function by its purpose - here, it is meant to draw the attention of the user